src/main/java/microsoft/exchange/webservices/data/util/TimeZoneUtils.java

```diff
@@ -30,576 +30,17 @@
 /**
  * Miscellany timezone functions
  */
-public class TimeZoneUtils {
+public final class TimeZoneUtils {
+
   //a map of olson name > Microsoft Name
-  final static private Map<String, String> olsonTimeZoneToMs = new HashMap<String, String>();
+  private final static Map<String, String> olsonTimeZoneToMs = createOlsonTimeZoneToMsMap();
```

Comment:
as mentioned on the [google java style](https://google-styleguide.googlecode.com/svn/trunk/javaguide.html#s4.8.7-modifiers) modifiers should be ordered in this way:
`public protected private abstract static final transient volatile synchronized native strictfp`